common/system/tasks/radio_manager.cpp

```diff
@@ -34,7 +32,7 @@ void radioManager_init() {
             sizeof(RadioRecievedPacket_s));
     hm_radioRegisterConsumer(i, &dataRx[i].rxBuffer);
 
-    strncpy(transmitPacket[i].callsign, call, 8);
+    strncpy(transmitPacket[i].callsign, cli_getConfigs()->callsign, 8);
```

Comment:
Will this always be 8?

Reply:
That was already there but you can check the update.